### DISCUSSION TITLE
Utility of Table of Contents Approach

### DISCUSSION THREAD

**Discussion by eyoung490** [CMSgov/price-transparency-guide#379] [Q&A]
While I understand the Table of Contents approach from the perspective of simplifying compliance in generating machine-readable files, I think it degrades their utility and could be contrary to the intention of the original rule. We are generating MRFs for TPAs who may have a relationship with a dozen or more rented networks, which they combine in various custom ways depending on the location (regional or for each satellite office, for example) and desired network quality of their plan sponsor clients. Furthermore, some plan sponsors may bundle in direct contracted rates for a certain provider group, or a third-party that handles reference-based pricing in some situations. If a plan sponsor uses several of these networks/negotiations and provides a ToC link to each Negotiated Rate file, as required by recent clarifications, a user of that file would still know nothing about claim routing between the several networks/negotiations that may exist for the same provider and item/service. Using context clues alone, it could be possible to understand the layering of these options without further information, but in others it would be impossible. Is the intention to leave this interpretation up to the user?

**Comment by themattrobinson** [comment]
Great question; I was wondering the same thing!

**Comment by urbanhipster** [comment]
Can you point me to the guidance allowing multiple in-network MRFs? We were under the impression there could only be one Negotiated Rate file per plan. Thank you!

**Reply by eyoung490** [reply]
@urbanhipster Although the plan sponsor will publish only one file, the Table of Contents file, several files will be referenced therein, according to the methodology summarized under the second FAQ on this page (https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification) titled "Can multiple in-network rate files be aggregated into a single in-network rate file?". So my question referenced the fact that several Negotiated Rate files published by others can be components of a single ToC MRF, and they can overlap in ways that would make it hard to determine which price was the price a patient would actually receive if they were to seek care from a certain provider for a certain procedure.

**Comment by shaselton-usds** [comment]
@eyoung490 Thanks for reaching out.

I'm struggling to understand this -- apologies! The original requirement was to produce a single file with all of the negotiated requirements found in the final rule. Clearly, this would lead to quite a bit of duplication not only in the files, but across files too.

Introducing the table-of-contents was a way to cut down the duplication while still capturing all of the same required information in the final rule. This put the work on the consumer of the files to "build" the single in-network file by combining the various in-network files found in the table-of-contents.

Where does the table-of-contents introduce a reporting plan's ambiguity that wasn't there with a single file? Thanks!

**Reply by urbanhipster** [reply]
Thank you for clarifying this. I thought the TOC was only for the opposite situation (when multiple plans share a single in-network file). But it sounds like we can also use it to leave the in-network files separate, rather than blending the data, when a plan uses multiple networks. That is a relief. We've been going around in circles on this for weeks, so thank you for the definitive explanation.

**Reply by CKreklow** [reply]
Perhaps I can help clarify.  Assume a situation where a plan has a network A, and Provider 1 is contracted with that network.  They also contract with a national network B for out-of-state coverage, and Provider 1 is contracted with that network as well.  If we assume that network B only publishes a single national In Network file, which is referenced by the plan's TOC, then Provider 1 will be in both the network A file and the network B file.

We have identified this situation as well, and are currently working to understand if we will need to modify the files we obtain from partner networks to remove the duplication and re-publish them, or if we will need to try to obtain filtered files from those networks.  Perhaps some type of a network hierarchy value within the TOC would be valuable to avoid requiring this additional effort while still providing clarity for the end users of the data?

Thanks.

**Reply by eyoung490** [reply]
@shaselton-usds Sorry, it looks like my original comment in reply to your question started its own thread. Does my explanation of the problems we see make sense? Does CMS have any proposal to restore the additional information that would have been communicated in the original negotiated rate file regarding the applicable rate for a given provider and procedure when more than one negotiated rate is shown across the several files referenced in the ToC approach for a single plan?

**Comment by eyoung490** [comment]
@shaselton-usds Perhaps I'm misunderstanding the construction of the TOC file. My understanding is that a TOC file is composed of links to Negotiated Rate files prepared by others representing all upstream negotiation arrangements for a given plan sponsor/plan combination. Say Plan A has a relationship with a regional provider network, a national provider network, a reference based price provider and has direct contracted rates with a local hospital/primary care group. Of the four relationships, the plan sponsor prepares a standard Negotiated Rate file only for the direct contracted rates then posts it online and references links to all four files in the Table of Contents file for Plan A. Provider B has price records for a given procedure in each of the four files, each price is different, and there are no rules explaining the priority of claims routing between them. Which price should they choose as the definitive negotiated rate for that service from Provider B?

**Reply by urbanhipster** [reply]
I agree -- whether the data is blended into one file or left separate, there will be a lot of situations where you have the same plan, the same procedure, the same provider, but different prices. I can only guess that one of the goals of transparency is to expose this kind of wackiness. Maybe in the long term there will be more standardization across the industry? I definitely think the immediate reaction once all this data becomes available will be confusion & chaos.  :D

**Reply by shaselton-usds** [reply]
@eyoung490 Thanks for the follow up. Definitely see the issue. I don't think introducing the ToC introduced this particular scenario for reporting due to the requirements of listing all providers (even if they are in multiple networks) to a negotiated rate -- it certainly is confusing if the same provider is part of many networks of which all have a different negotiated rate. The Allowed Amounts file was meant to provide this type of retrospective clarity vs what is actually in the contracts (the in-network file). This type of scenario is handled at the time of adjudication of an actual claim and the in-network file it disclosing data that is further upstream. 

Aside from that - what is the typical resolution during adjudication for your company when this happens? It might be worth trying to introduce a concept of "weights" or "priorities" in future iterations.

**Reply by eyoung490** [reply]
@shaselton-usds Under the original file structure, we could have specified the only applicable rate for a certain service and provider, even if several networks had negotiated rates, because we have transparency into the priority of the networks for any given service and provider. ToC made it impossible to communicate that level of detail.

Having said that, a network priority file being included in the ToC file would probably resolve some simple cases. Consider the health plan of a large hospital for example. In our experience, plans like this can safely stack 100% of their special pricing for their own employees ahead of the pricing from their larger rented medical network. More complex special arrangements and plans with three or more integrated networks are a total toss up though and the ToC format file couldn't be used to power a transparency tool that accurately reflects negotiated rates 100% of the time.

**Reply by CKreklow** [reply]
> The Allowed Amounts file was meant to provide this type of retrospective clarity vs what is actually in the contracts (the in-network file). This type of scenario is handled at the time of adjudication of an actual claim and the in-network file it disclosing data that is further upstream.

This seems like a misleading comment, because the Allowed Amounts file is for Out-of-Network only, so there was never an intent to provide "retrospective clarity" for In-Network rates.
